src/scanner.cpp

```diff
@@ -311,14 +314,15 @@ LaserScan Scanner::getCompleteScan()
       exception_occured = false;
       try
       {
-        monitoring_frame = fetchMonitoringFrame();
+        monitoring_frame = fetchMonitoringFrame(fetch_frame_timeout);
         parseFields(monitoring_frame);
         isDiagnosticInformationOk(monitoring_frame.diagnostic_area_.diagnostic_information_);
       }
       catch (const FetchMonitoringFrameException& e)
       {
         std::cerr << e.what() << '\n';
         exception_occured = true;
+        fetch_frame_timeout = adjustTimeout(fetch_frame_timeout);
```

Comment:
Without a namespace it may take a while to understand where this function is from.
E.g. `utils::adjustTimeout` could add clarity.

Reply:
Done.